extensions/keepassxc/package.json

```diff
@@ -48,6 +48,14 @@
       "description": "KeepassXC.app",
       "placeholder": "Select KeepassXC.app",
       "default": "/Applications/KeePassXC.app"
+    },
+    {
+      "name": "yubikey",
+      "type": "textfield",
+      "required": false,
+      "title": "Yubikey Slot",
+      "description": "Yubikey slot and optional serial used to",
```

Comment:
It is recommended to give a specific example value in the description.

Reply:
eg: `1` or `1:1`?

Reply:
> I don't have a device to test this feature, please make sure that it has been tested and works fine.

Hi,

The fact that we need to touch the device each time is a bit cumbersome so I don't thing the PR worth to be merged as it is, the solution were we use the cli to open the database and kept the reference to the opened cli to make research and have an action to close/release cli will be better.

also sometime if the touch has been made to fast or to slow  it create an exception and we need to restart the raycast action. So if  we have the option  to wait until it's successfully opened with good feedback/retry should be considered.

But I thing a good support of keepassXC in Raycast will be very nice.

For now I have sync low critical password to a less secure database so I can I use the plugin as it is.